lib/tasks/style_checker.rake

```diff
@@ -0,0 +1,48 @@
+RUBY = /\.(rb)|(rake)$/
+JS = /\.jsx?$/
+
+desc 'Style checks files that differ from master'
+task :check_style do
+  puts diff_output
+  puts check_ruby
+  puts check_js
+  exit evaluate
+end
+
+def evaluate
+  return 0 if check_ruby.include? 'no offenses'
+  return 0 if check_js.include? 'true'
+  1
+end
+
+def diff_output
+  "Files found in the diff\n#{diff}\n"
+end
+
+def check_ruby
+  files = files_that_match RUBY
+  return 'No ruby files found!' if files.empty?
+  @ruby_results ||= "#{rubocop(files)}\n#{system rubocop(files)}\n"
+end
+
+def check_js
+  files = files_that_match JS
+  return 'No javascript files found!' if files.empty?
```

Comment:
Wouldn't this cause `evaluate` to return `1` since it doesn't include the word `true`?